specification.rst

```diff
@@ -2854,3 +2854,21 @@ The page abbreviation rules for the different values of the
 "minimal-two"
     As "minimal", but at least two digits are kept in the second number when it
     has two or more digits long.
+
+
+Appendix VI: Links
+------------------
+
+The CSL syntax does not have support for configuration of links. However,
+processors should include links on bibliographic references, using the following
+rules:
+
+If the item includes the following identifiers, the target of the link should be
+set, in order of priority:
```

Comment:
This assumes only one URL is printed in the output. I suppose if we want to retain this restriction, we could make that explicit in this text.

Reply:
We could say, "If there is only one identifier, link the whole reference. If there are identifiers, link them individually."